src/main/scala/intent/core/expectations/contain.scala

```diff
@@ -5,7 +5,7 @@ import scala.concurrent.Future
 import scala.collection.mutable.ListBuffer
 import scala.Array
 
-private def evalToContain[T](actual: IterableOnce[T],
+private def evalToContain[T](actual: IterableOnce[T] | Null,
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.

**Nullable variable is used**
Requires `.nn` to remove nullness. | 1